manifests/init.pp

```diff
@@ -67,6 +76,9 @@
   Stdlib::Absolutepath $webserver_static_dir = '/var/lib/pulp/docroot',
   String $postgresql_db_name = 'pulpcore',
   String $postgresql_db_password = extlib::cache_data('pulpcore_cache_data', 'db_password', extlib::random_password(32)),
+  Stdlib::Host $postgresql_db_host = '127.0.0.1',
```

Comment:
Probably better to use `localhost` since we're slowly leaving the IPv4 world behind.
```suggestion
  Stdlib::Host $postgresql_db_host = 'localhost',
```